docker/root/etc/s6/openssh/setup

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
-
-if [ ! -d /data/ssh ]; then
-    mkdir -p /data/ssh
-fi
-
-if [ ! -f /data/ssh/ssh_host_ed25519_key ]; then
-    echo "Generating /data/ssh/ssh_host_ed25519_key..."
-    ssh-keygen -t ed25519 -f /data/ssh/ssh_host_ed25519_key -N "" > /dev/null
-fi
-
-if [ ! -f /data/ssh/ssh_host_rsa_key ]; then
-    echo "Generating /data/ssh/ssh_host_rsa_key..."
-    ssh-keygen -t rsa -b 2048 -f /data/ssh/ssh_host_rsa_key -N "" > /dev/null
-fi
-
-if [ ! -f /data/ssh/ssh_host_dsa_key ]; then
-    echo "Generating /data/ssh/ssh_host_dsa_key..."
-    ssh-keygen -t dsa -f /data/ssh/ssh_host_dsa_key -N "" > /dev/null
-fi
-
-if [ ! -f /data/ssh/ssh_host_ecdsa_key ]; then
-    echo "Generating /data/ssh/ssh_host_ecdsa_key..."
-    ssh-keygen -t ecdsa -b 256 -f /data/ssh/ssh_host_ecdsa_key -N "" > /dev/null
-fi
-
-chown root:root /data/ssh/*
-chmod 0700 /data/ssh
-chmod 0600 /data/ssh/*
```

Comment:
The ssh-hostkeys are owned by root and locked down to root only read access. The internal ssh-server will not be able to read them as git user.